connection-loader/src/main/java/com/terracotta/connection/api/DetailedConnectionException.java

```diff
@@ -0,0 +1,33 @@
+package com.terracotta.connection.api;
+
+import org.terracotta.connection.ConnectionException;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/*
+  Should this class be going to connection-api project?
```

Comment:
I think this should go into connection-api

Reply:
To Myron,

Please share your thoughts on this.

Reply:
I don’t think anyone is is going to catch this exception specifically so probably ok here